addons/web/static/src/core/utils/strings.js

```diff
@@ -8,9 +8,10 @@ export const escapeMethod = Symbol("html");
  * Escapes a string for HTML.
  *
  * @param {string | number} [str] the string to escape
+ * @param {boolean} [skipQuotes=false] - whether to skip escaping single, double and back quotes. Default is false.
  * @returns {string} an escaped string
  */
-export function escape(str) {
+export function escape(str, skipQuotes = false) {
```

Comment:
We should use an options object so we can know which option is passed:

`export function escape(str, options = { skipQuotes: false })`

When you use this function you have to pass named option, it make the code easier to read.